src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -276,6 +289,8 @@ internal Exception Abort(Exception abortException)
             // Stop sending requests to this connection.
             _pool.InvalidateHttp3Connection(this);
 
+            WebtransportManager?.Dispose();
```

Comment:
Why do you think you need to dispose here?

Reply:
I thought that aborting the connection will give us another way of closing it but looking better at the code I realised that the connection is not nulled so I will erase that since it will enter inside the CheckForShutdown anyway